### PR TITLE
Filter args to removed empty or blank args

### PR DESCRIPTION
**Pull request by binaryoverload**

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/Events.java [hunk 1]
```diff
@@ -290,7 +292,7 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
                                 return;
                             COMMAND_COUNTER.computeIfAbsent(event.getChannel().getGuild().getId(),
                                     g -> new AtomicInteger()).incrementAndGet();
-                            String[] finalArgs = args;
+                            String[] finalArgs = Arrays.stream(args).filter(StringUtils::isNotBlank).toArray(String[]::new);
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/Events.java]: No need using StringUtils String isEmpty works fine here

**Comment by binaryoverload** [comment]
@ArsenArsen the travis .yml has been removed . I was just experimenting with it

**Comment by binaryoverload** [comment]
@ArsenArsen the travis .yml has been removed . I was just experimenting with it

**Comment by binaryoverload** [comment]
@ArsenArsen the travis .yml has been removed . I was just experimenting with it

**Comment by ArsenArsen** [comment]
EDIT: Did you test the changes?

**Comment by binaryoverload** [comment]
Changed after testing

**Comment by ArsenArsen** [comment]
Oh I can't read. Sorry for that. Merging

**Pull request merged**